docker/cloudbuild.yaml

```diff
@@ -127,6 +129,12 @@ steps:
         docker push $$DOCKER_REPO:$COMMIT_SHA
         docker tag $$SENTRY_IMAGE $$DOCKER_REPO:nightly
         docker push $$DOCKER_REPO:nightly
+        docker tag $$SENTRY_IMAGE $$DOCKER_REPO:$SHORT_SHA-py2
+        docker push $$DOCKER_REPO:$SHORT_SHA-py2
+        docker tag $$SENTRY_IMAGE $$DOCKER_REPO:$COMMIT_SHA-py2
+        docker push $$DOCKER_REPO:$COMMIT_SHA-py2
+        docker tag $$SENTRY_IMAGE $$DOCKER_REPO:nightly-py2
+        docker push $$DOCKER_REPO:nightly-py2
```

Comment:
So `-py2` is an alias for the default?

Reply:
Yes as per the plan in the PR description, we are going to introduce a `SENTRY_PYTHON2` option for folks wanting to stay on Py2. For that to work, we need the `-py2` suffix. After that, we will make the default images to use PY3 and remove the `-py3` variants (as that will be the default).